src/script/interpreter.cpp

```diff
@@ -1975,10 +1976,10 @@ CConfidentialValue TransactionSignatureChecker::GetValueInPrevIn() const
     return amountPreviousInput;
 }
 
-bool TransactionSignatureChecker::IsConfirmedBitcoinBlock(const uint256& genesishash, const uint256& hash, bool fConservativeConfirmationRequirements) const
+bool TransactionSignatureChecker::IsConfirmedBitcoinBlock(const uint256& genesishash, const uint256& hash, bool fConservativeConfirmationRequirements, uint32_t nConfirmationsRequired) const
 {
 #ifndef BITCOIN_SCRIPT_NO_CALLRPC
-    return ::IsConfirmedBitcoinBlock(genesishash, hash, fConservativeConfirmationRequirements ? 10 : 8);
+    return ::IsConfirmedBitcoinBlock(genesishash, hash, nConfirmationsRequired + (fConservativeConfirmationRequirements ? 2 : 0));
```

Comment:
Perhaps we should consider removing SCRIPT_VERIFY_INCREASE_CONFIRMATIONS_REQUIRED and fConservativeConfirmationRequirements ? Perhaps I should just read more about this...
At most one constant with a 2 seems enough.

Reply:
Let's leave this for another PR I guess,